net-libs/libotr/libotr-4.1.1.ebuild

```diff
@@ -17,6 +17,10 @@ RDEPEND="
 	dev-libs/libgpg-error:0="
 DEPEND="${RDEPEND}"
 
+PATCHES=(
+	"${FILESDIR}/fix_build_with_libgcrypt_1.10.patch"
```

Comment:
I don't think it's explicitly stated in the devmanual, but maybe kebab-case the name instead of snake_case?

Reply:
Did a (not very thorough) scan of ::gentoo

```
$ ~/sources/gentoo/gentoo (eapi8/x11-plugins/gkrellm+*?) $ find -name '*.patch' |grep '_' |wc -l
1798
$ ~/sources/gentoo/gentoo (eapi8/x11-plugins/gkrellm+*?) $ find -name '*.patch' |grep '-' |wc -l
12020
```
I guess it's just what people prefer :-)

Reply:
also prefix with P in filename

Reply:
> I don't think it's explicitly stated in the devmanual, but maybe kebab-case the name instead of snake_case?

I have never used snake_case before in patches, maybe I should go and lie down for a bit :)